samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalyzer/FaceSentimentAnalyzerSkill.cpp

```diff
@@ -71,8 +92,20 @@ namespace winrt::Contoso::FaceSentimentAnalyzer::implementation
         auto faceDetector = FaceDetector::CreateAsync().get();
 
         // Load WinML model
-        auto modelFile = Windows::Storage::StorageFile::GetFileFromApplicationUriAsync(Windows::Foundation::Uri(L"ms-appx:///Contoso.FaceSentimentAnalyzer/" + WINML_MODEL_FILENAME)).get();
         
+        winrt::Windows::Storage::StorageFile modelFile = nullptr;
+        if (IsUWPContainer())
+        {
+            modelFile = Windows::Storage::StorageFile::GetFileFromApplicationUriAsync(Windows::Foundation::Uri(L"ms-appx:///Contoso.FaceSentimentAnalyzer/" + WINML_MODEL_FILENAME)).get();
+        }
+        else
+        {
+            WCHAR DllPath[MAX_PATH] = { 0 };
```

Comment:
>0  [](start = 40, length = 2)

Remove 0? Per Anders in another PR mentioned for structure (not array) - {The extra "0" just causes the first member in the structure to be set to 0 explicitly, in a separate step, and then we clear the rest of the structure. Might just as well clear the whole structure in a single shot.}

Reply:
This is a char array used as a string, we just need a null termination, and don't need to zero the entire memory